content/300-guides/200-deployment/110-deployment-guides/250-deploying-to-koyeb.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: 'Deploying to Koyeb'
+metaTitle: 'Deploy a Prisma app to Koyeb'
+metaDescription: 'Learn how to deploy a Node.js server that uses Prisma to Koyeb Serverless Platform.'
+---
+
+<TopBlock>
+
+In this guide, you will set up and deploy a Node.js server that uses Prisma with PostgreSQL to [Koyeb](https://www.koyeb.com). The application exposes a REST API and uses Prisma Client to handle fetching, creating, and deleting records from a database.
+
+Koyeb is a developer-friendly serverless platform to deploy apps globally. The platform lets you seamlessly run Docker containers, web apps, and APIs with git-based deployment, TLS encryption, native autoscaling, a global edge network, and built-in service mesh & discovery.
+
+When using the [Koyeb git-driven deployment](https://www.koyeb.com/docs/apps/build-from-git) method, each time you push code changes to a GitHub repository a new build and deployment of the application are automatically triggered on the Koyeb Serverless Platform.
+This guide uses the latter approach whereby you push your code to the app's repository on GitHub.
+
+The application has the following components:
+
+- **Backend**: Node.js REST API built with Express.js with resource endpoints that use Prisma Client to handle database operations against a PostgreSQL database (e.g., hosted on PlanetScale).
```

Comment:
```suggestion
- **Backend**: Node.js REST API built with Express.js with resource endpoints that use Prisma Client to handle database operations against a relational database (e.g., hosted on PlanetScale).
```

Here it kind of implies that PlanetScale is running on PostgreSQL but they are actually MySQL compatible